frontend/javascripts/oxalis/view/right-border-tabs/starting_job_modals.tsx

```diff
@@ -315,3 +433,111 @@ export function NeuronInferralModal({ handleClose }: Props) {
     />
   );
 }
+
+type MaterializeVolumeAnnotationModalProps = Props & {
+  selectedVolumeLayer?: APIDataLayer;
+};
+
+export function MaterializeVolumeAnnotationModal({
+  selectedVolumeLayer,
+  handleClose,
+}: MaterializeVolumeAnnotationModalProps) {
+  const dataset = useSelector((state: OxalisState) => state.dataset);
+  const tracing = useSelector((state: OxalisState) => state.tracing);
+  const activeSegmentationTracingLayer = useSelector(getActiveSegmentationTracingLayer);
+  const fixedSelectedLayer = selectedVolumeLayer || activeSegmentationTracingLayer;
+  const isMergerModeEnabled = useSelector(
+    (state: OxalisState) => state.temporaryConfiguration.isMergerModeEnabled,
+  );
+  let description = (
+    <p>
+      Start a job that takes the current state of this volume annotation and materializes it into a
+      new dataset. All annotations done on the
+      {` "${fixedSelectedLayer && getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing)}" `}
+      volume layer will be merged with the data of the fallback layer.
+      {isMergerModeEnabled
+        ? " Additionally, the skeletons will be used to merge segments as merger mode is active. "
+        : " "}
+      Please enter the name of the output dataset and the output segmentation layer.
+    </p>
+  );
+  if (tracing.volumes.length === 0) {
+    description = (
+      <p>
+        Start a job that takes the current state of this merger mode tracing and materializes it
+        into a new dataset. In the new output dataset the segments connected via skeleton nodes will
+        be merged. Please enter the name of the output dataset and the output segmentation layer.
+      </p>
+    );
+  }
```

Comment:
It would be nice if you could check whether this text makes sense to you @daniel-wer.

The reassignment of `description` in the if only happens when there is no volume annotation (and therefore the annotation is skeleton-only).

> `` "${fixedSelectedLayer && getReadableNameOfVolumeLayer(fixedSelectedLayer, tracing)}" ``

Here the `fixedSelectedLayer` is needed because `fixedSelectedLayer` is potentially undefined if there is no volume annotation layer. That's why I added the reassignment of `description` in the if statement to cover this case and give a better text to the user that matches the context.

Reply:
Instead of having two different ways to describe merger mode you could replace

> Additionally, the skeletons will be used to merge segments as merger mode is active.

and this part of the description

> In the new output dataset the segments connected via skeleton nodes will be merged.

both with 

"Since merger mode is currently active, in the new output dataset the segments connected via skeleton nodes will be merged."

but it's fine like this as well.